content/en/monitors/create/configuration.md

```diff
@@ -120,18 +120,50 @@ See the documentation for [process check][1], [integration check][2], and [custo
 
 **Note**: It is recommended that you set the missing data window to at least two times the evaluation period.
 
-Alternatively, if you are monitoring a metric over an auto-scaling group of hosts that stop and start automatically, notifying for no data would produce a lot of notifications. For this case, you should not enable notifications for missing data. This option does not work if it is enabled at a time when data has not been reporting for a long period.
+If you are monitoring a metric over an auto-scaling group of hosts that stop and start automatically, notifying for no data would produce a lot of notifications. For this case, you should not enable notifications for missing data. This option does not work if it is enabled at a time when data has not been reporting for a long period.
 
 ##### Grouping
 
 For a monitor that does not notify on missing data, if a group does not report data, the monitor skips evaluations and eventually drops the group. During this period, the bar in the results page stays green. When there is data and groups start reporting again, the green bar shows an OK status and backfills to make it look like there was no interruption.
 
+{{% /tab %}}
+
+{{% tab "Other monitor types" %}}
+
+{{< img src="/monitors/create/on_missing_data.png" alt="No Data Options" style="width:90%;">}}
+
+If data is missing for `N` minutes: `Evaluate as zero` / `Show last known status`, `Show NO DATA`, `Show NO DATA and notify`, or `Show OK`.
+
+The selected behaviour will be applied once a monitor's query does not return any data. Hence, contrary to the `Do not notify` option, the missing data window is **not** configurable.
+
+| Option                    | Monitor status & notification                                             |
+|---------------------------|---------------------------------------------------------------------------|
+| `Evaluate as zero`        | Empty result is "replaced" with zero and compared to the alert/warning thresholds. For example, if the alert threshold is set to `> 10`, a zero would not trigger that condition, hence the Monitor status is set to `OK`.   |
+| `Show last known status`  | The last known status of the group/monitor is set.                        |
+| `Show NO DATA`            | Monitor status is set to `NO DATA`.                                       |
+| `Show NO DATA and notify` | Monitor status is set to `NO DATA` and a notification is sent out.        |
+| `Show OK`                 | Monitor is resolved and status is set to `OK`.                            |
+
+The options `Evaluate as zero` and `Show last known status` are shown depending on the type of the query. `Evaluate as zero` is shown for monitors using `Count` queries. `Show last known status` on the other hand is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, or `Distribution`. These options are considerd as the **default status** of the monitor.
```

Comment:
I would recommend to make two sections with bold title to illustrate the two cases.